pyproject.toml

```diff
@@ -80,11 +81,15 @@ dev = [
     "time-machine >= 2.15.0, < 3", # Used in test_signers.py
     "tomli; python_version<'3.11'", # Used in test_version.py
 ]
+httpx = [
```

Comment:
Why do we need this as a dependency group?

Reply:
does not work ... and after messing around in a separate project I'm growing skeptical that the current setup is working either.
I don't understand the venv management of uv at all

... ugh the CI thing isn't even running the right step

Reply:
I got it working now, but I'm still confused why some of the stuff I did failed.